identity-hub-client/src/main/java/org/eclipse/dataspaceconnector/identityhub/client/ApiException.java

```diff
@@ -0,0 +1,50 @@
+/*
+ *  Copyright (c) 2022 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Microsoft Corporation - initial API and implementation
+ *
+ */
+
+package org.eclipse.dataspaceconnector.identityhub.client;
+
+import okhttp3.Headers;
+import okhttp3.ResponseBody;
+
+/**
+ * Base exception for IdentityHub rest client, used to represent api errors. 
+ */
+public class ApiException extends RuntimeException {
```

Comment:
Can this extend EdcException?

Reply:
I can change it, then the IdentityClient would depend on edc core-spi.
Can you explain what would be the value?